ibutex.py

```diff
@@ -63,8 +66,14 @@
 
 if not args.quick:
 	subprocess.call(fullbib)
-	subprocess.call(fullcmd)
-	subprocess.call(fullcmd)
+	success = 0
+	if args.sagemath:
+		success = subprocess.call(fullsage)
+	if success != 0:
```

Comment:
indent this by one, then you can omit the `success = 0` above and if this is a fatal error anyways, simply `exit(4)` here, equivalent to the first `fullcmd` call above. This has two advantages, firstly we do less branching, secondly fatalities are more obvious.

Reply:
fixed with newest commit